core/src/action.rs

```diff
@@ -100,6 +100,9 @@ pub enum Action {
     /// Does not validate, assumes link is valid.
     AddLink(Link),
 
+    //Removes a link for the local DHT
+    RemoveLink(Link),
```

Comment:
Last I heard about removing links (a few weeks ago), I thought it had to reference the hash of a previously issued `AddLink`. If so, shouldn't that be part of the action?

Reply:
Per heartbeat, we decided we don't need this for closed alpha and can go forward with this implementation